### ISSUE TITLE
Add_vpool extend failed `ArakoonException: Unknown error code 0x20200a7b, server said: {"volatile": "memcache", "persistent": "pyrakoon"}`

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#946]
During vpool extending I observed this exception on the setup.
This exception caused the vPool status to be stuck in `EXTENDING`. 
https://github.com/openvstorage/framework/blob/master/ovs/lib/storagerouter.py#L1009

```
2016-09-22 18:03:03 30500 +0200 - stor-01.be-g8-4 - 21400/139928115660608 - celery/celery.worker.job - 1846 - ERROR - Task ovs.storagerouter.add_vpool[e7272337-5e9c-4c1f-806c-4c5728f41f35] raised unexpected: ArakoonException('Unknown error code 0x20200a7b, server said: {"volatile": "memcache", "persistent": "pyrakoon"}',)
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/OpenvStorage/ovs/lib/storagerouter.py", line 1009, in add_vpool
    storagedriver_config.load()
  File "/opt/OpenvStorage/ovs/extensions/storageserver/storagedriver.py", line 257, in load
    self.configuration = json.loads(Configuration.get(self.key, raw=True))
  File "/opt/OpenvStorage/ovs/extensions/generic/configuration.py", line 112, in get
    data = Configuration._get(key_entries[0], raw)
  File "/opt/OpenvStorage/ovs/extensions/generic/configuration.py", line 362, in _get
    key=key)
  File "/opt/OpenvStorage/ovs/extensions/generic/configuration.py", line 404, in _passthrough
    return getattr(ArakoonConfiguration, method)(*args, **kwargs)
  File "/opt/OpenvStorage/ovs/extensions/db/arakoon/configuration.py", line 119, in get
    return client.get(key)
  File "/opt/OpenvStorage/ovs/extensions/db/arakoon/pyrakoon/client.py", line 44, in new_function
    return f(self, *args, **kw)
  File "/opt/OpenvStorage/ovs/extensions/db/arakoon/pyrakoon/client.py", line 76, in get
    return PyrakoonClient._try(self._identifier, self._client.get, key, consistency)
  File "/opt/OpenvStorage/ovs/extensions/db/arakoon/pyrakoon/client.py", line 207, in _try
    return_value = method(*args, **kwargs)
  File "<update_argspec>", line 5, in get
  File "/opt/OpenvStorage/ovs/extensions/db/arakoon/pyrakoon/pyrakoon/compat.py", line 152, in wrapped
    raise new_exception
ArakoonException: Unknown error code 0x20200a7b, server said: {"volatile": "memcache", "persistent": "pyrakoon"}
```

**Comment by khenderick** [comment]
The high error code seems like the client received data that should have been send to another client. That kinda sounds like this env is running < openvstorage-2.7.3-rev.4013.c4ac0a4 or that there's some bug related to #942 left where the client is still sharing some socket.

**Comment by khenderick** [comment]
After discussion with @saelbrec, it turns out that `2.7.3-rev.4011.a41bbc3-1` was installed, which did not contains the fix of #942, so most likely this duplicates that ticket. I'll just put it in state_verification anyway.

**Comment by JeffreyDevloo** [comment]
### Information
Following the hypothesis that I would be because of Pyrakoons errors and the fact that we have not encountered a similar issue anymore in our automated tests, I am inclined to close this ticket.
